_posts/2015-02-21-Test-Jammin.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "Test Jam"
+post\_title\_id:  "testjam"
+date:   2015-02-21
+author: orta
+categories: event virtual cocoapods tests 
+---
+
+Last year, around this time, we ran a [Bug Bash][1] over a weekend. It went beautifully. We triaged a whole lot of issues, got a lot of minor issues fixed and found some new members for the core team.
+
+Not ones to repeat ourselves, we’ve begun preparations on the next big community event. It’s a test jam, and everyone is invited.
+
+*TLDR/Elevator Pitch*: If the community is going to move towards Swift, then let’s make sure that our Objective-C is in order by adding tests to established libraries.
+
+Find out more below.
+
+<!-- more -->
+
+Date: A weekend in late April / early May
+Locations: NYC / SF / + More
+
+We have picked out 200 pods without tests, or with very low test coverage, ordered by [popularity][2]. These are assigned randomly to interested contributors via an IRC bot. They then will aim to:
```

Comment:
I think we're moving very quickly here. We should probably explain what IRC is first.